open-hackathon-server/src/hackathon/views/schema/HackathonNoticeListResource.json

```diff
@@ -0,0 +1,42 @@
+{
+  "HackathonNoticeListResource": {
+    "get": {
+      "input": {
+        "description": "list hackathon notices",
+        "type": "object",
+        "properties": {
+          "hackathon_name": {
```

Comment:
HackathonNoticeListResource.get is not hackathon_name_required
We can list all notices with this API.